### ISSUE TITLE
Game crashes quite often

### ISSUE THREAD

**Issue by ErlkoenigWJ** [SolastaMods/SolastaUnfinishedBusiness#4575]
While playing (especially in combat) the game often says it has detected an instability and then crashes to desktop. It also happens when I attempt to load a save file. I've attached the player log. Please let me know if there's any other info I can provide.

[Player.log](https://github.com/SolastaMods/SolastaUnfinishedBusiness/files/15203266/Player.log)

**Comment by ThyWoof** [comment]
Are you on steam, gog or windows store? Before I help further please share a screenshot of Solasta in one of these cloud vendors. Your log has all indication of a pirate copy of the game.

**Comment by ErlkoenigWJ** [comment]
If so, I assume the indication's not really reliable. Let me know if you also need a picture from the browser version.

<img width="1912" alt="solasta_galaxy" src="https://github.com/SolastaMods/SolastaUnfinishedBusiness/assets/68244623/b0c1c27c-d02a-4933-ba19-41e8f2cff460">

**Comment by ErlkoenigWJ** [comment]
Sorry, I don't know which save caused the last one anymore, but I got a new crash and attached both the log and the save this time.

[Player.log](https://github.com/SolastaMods/SolastaUnfinishedBusiness/files/15205469/Player.log)
[QuickSave_14.zip](https://github.com/SolastaMods/SolastaUnfinishedBusiness/files/15205470/QuickSave_14.zip)

**Comment by ThyWoof** [comment]
![image](https://github.com/SolastaMods/SolastaUnfinishedBusiness/assets/13087646/85f5f7e8-1c3a-4049-8e15-65cc885cb454)
I have no issues loading this save. Please share the mod settings.xml file. You can export it from mod UI.

**Comment by ThyWoof** [comment]
based
[SolastaUnfinishedBusiness.zip](https://github.com/SolastaMods/SolastaUnfinishedBusiness/files/15205715/SolastaUnfinishedBusiness.zip)
I did a small tweak based on the log. Unzip this and replace the DLL under mods/SolastaUnfinishedBusiness. Let me know if you can load.

**Comment by ErlkoenigWJ** [comment]
Loading the save file works 1 out of 3 times for me as well, but after playing for a while it'll say something along the lines of "instability detected" and crash. It's not a fully deterministic thing, unfortunately. The new dll doesn't seem to have solved it. 
I've attached the mod settings just in case.

[M_Erlkonig.zip](https://github.com/SolastaMods/SolastaUnfinishedBusiness/files/15205948/M_Erlkonig.zip)

**Comment by ThyWoof** [comment]
Even with your mod settings I have no issue opening the game. There might be some racing conditions if you have a slow gig or game files not on a SSD. Unfortunately I cannot fix as I cannot repro.

**Comment by ErlkoenigWJ** [comment]
I know this is closed, but in case it's useful in the future, I've been able to trace the issue to the great weapon master feat. Activating that power or even just picking the feat often causes instability messages, and afterwards the trouble with random save loading instabilities or just random instabilities in combat in general starts. I've done two playthroughs up to lvl 4 and had no issues unless I picked GWM.
